[2024-05-18T19:04:46.604+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T19:04:46.624+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2022-10-29T01:00:00+00:00 [queued]>
[2024-05-18T19:04:46.631+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2022-10-29T01:00:00+00:00 [queued]>
[2024-05-18T19:04:46.632+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-18T19:04:46.646+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hello_task> on 2022-10-29 01:00:00+00:00
[2024-05-18T19:04:46.652+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=490) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-18T19:04:46.653+0000] {standard_task_runner.py:63} INFO - Started process 492 to run task
[2024-05-18T19:04:46.654+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_loader', 'hello_task', 'scheduled__2022-10-29T01:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmpjw71toiy']
[2024-05-18T19:04:46.656+0000] {standard_task_runner.py:91} INFO - Job 95: Subtask hello_task
[2024-05-18T19:04:46.700+0000] {task_command.py:426} INFO - Running <TaskInstance: data_loader.hello_task scheduled__2022-10-29T01:00:00+00:00 [running]> on host b5cbc22da81e
[2024-05-18T19:04:46.772+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_loader' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2022-10-29T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-29T01:00:00+00:00'
[2024-05-18T19:04:46.774+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T19:04:46.775+0000] {logging_mixin.py:188} INFO - 2022-10-29/
[2024-05-18T19:04:46.775+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-18T19:04:46.775+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T19:04:46.781+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_loader, task_id=hello_task, run_id=scheduled__2022-10-29T01:00:00+00:00, execution_date=20221029T010000, start_date=20240518T190446, end_date=20240518T190446
[2024-05-18T19:04:46.829+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-18T19:04:46.845+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-18T19:04:46.849+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
